tools/scconfigconverter.py

```diff
@@ -0,0 +1,76 @@
+import csv
```

Comment:
add python header to allow run correct python without adding binary before script
for python without version can be: ``#! /usr/bin/env python``
or for python2: ``#! /usr/bin/env python2``
And also script should be with executable flag if it not set